mypy/typeshed/stdlib/pathlib.pyi

```diff
@@ -41,6 +41,7 @@ class PurePath(PathLike[str]):
     def __new__(cls, *args: StrPath) -> Self: ...
     def __hash__(self) -> int: ...
     def __eq__(self, other: object) -> bool: ...
+    def __str__(self) -> str: ...
```

Comment:
we should just get rid of basedtypeshed since it seems easier to just modify the vendored one

Reply:
yeah